README.md

```diff
@@ -53,7 +53,7 @@ WordPress.org-specific customizations for the Two Factor plugin
 1. Activate all four plugins.
 1. If you want to make JS changes, then `cd settings && npm start`
 1. Open `wp-admin/options-general.php?page=bbpress` and uncheck `Prefix all forum content with the Forum Root slug (Recommended)`, then save.
-1. Visit https://example.org/users/{username}/edit/account/ to view the custom settings UI. If you get a `404` error, visit `wp-admin/options-permalinks.php` and then try again.
+1. Visit https://example.org/users/{username}/edit/account/ to view the custom settings UI. If you get a `404` error, visit `wp-admin/options-permalinks.php`, update Permalinks to `Post name`, save and then try again.
```

Comment:
Mine is set at `Day and name`, but this should also work. What was your set to by default? Unless it was `Plain`, then just visiting the page may have been what fixed it, since [that flushes the rewrite rules even if you don't save](https://github.com/WordPress/WordPress/blob/1bf93a87a449a9476d1efd93a0452d7d6ceb3808/wp-admin/options-permalink.php#L212).

If that's what fixed it, then we could update the readme to better explain that.

Reply:
Ah, I stood the environment up with #76. As [you mention](https://github.com/WordPress/wporg-two-factor/pull/76/files#r1131468391), this step should be added there instead. 

Just visiting doesn't work in that context, saving the permalinks doesn't either. They need to be changed?
https://d.pr/i/mpAAkI

Reply:
It looks like they're originally set to `Plain` in that gif, but I don't think rewrite rules work at all with plain permalinks. So it makes sense that they'd need to be saved in that situation.

IMO the env should setup "pretty" permalinks when it's created. cc @dd32

Reply:
Yeah true.